src/angles.jl

```diff
@@ -8,17 +8,14 @@
 Angle ∠ABC between rays BA and BC.
 See https://en.wikipedia.org/wiki/Angle.
 
+Uses the two-argument form of `atan`.
+See https://en.wikipedia.org/wiki/Atan2.
+
 ## Example
 
 ```julia
 ∠(Point(0,1), Point(0,0), Point(1,0)) == π/2
```

Comment:
Can you also fix the typo in this example? We should swap the first and third points in order to get a positive pi/2